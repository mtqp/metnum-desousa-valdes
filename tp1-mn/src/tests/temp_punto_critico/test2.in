400 400 80 50 400 1
300.0 150.0