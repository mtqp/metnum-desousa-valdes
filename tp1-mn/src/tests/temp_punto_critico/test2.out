0	0	-100.00000
0	1	-100.00000
0	2	-100.00000
0	3	-100.00000
0	4	-100.00000
0	5	-100.00000
1	0	-100.00000
1	1	-77.43521
1	2	-45.93387
1	3	2.32350
1	4	50.58088
1	5	-100.00000
2	0	-100.00000
2	1	-63.80697
2	2	-8.62377
2	3	104.64701
2	4	400.00000
2	5	-100.00000
3	0	-100.00000
3	1	-69.16890
3	2	-29.40125
3	3	24.88829
3	4	70.24129
3	5	-100.00000
4	0	-100.00000
4	1	-83.46738
4	2	-64.70063
4	3	-45.93387
4	4	-43.92315
4	5	-100.00000
5	0	-100.00000
5	1	-100.00000
5	2	-100.00000
5	3	-100.00000
5	4	-100.00000
5	5	-100.00000
